src/pages/Reviews.jsx

```diff
@@ -5,20 +5,22 @@ import './reviews.css'
 
 export default function Reviews() {
     
-    const [reviews, setReviews] = useState([])
+    const [reviews, setReviews] = useState([]);
+    const [isLoading, setIsLoading] = useState(true);
 
     useEffect(() => {
```

Comment:
Please make sure that the first line of your useEffects where there is a loading pattern is `setIsLoading(true)`